app/views/partials/json-schema/properties.hbs

```diff
@@ -42,16 +43,21 @@
             {{>json-schema/datatype discriminator=(equal @key ../discriminator)}}
           </td>
         --}}
+        {{! The Description column }}
         <td>
-          {{! The Description column }}
-          {{#if description}}
-            {{md description}}
-            {{!-- Show details of nested property schema?
-              <div class="json-inner-schema">
-                {{>json-schema/body $ref="" description=""}}
-              </div>
-            --}}
-          {{/if}}
+          {{ md (
+            concat
+              (ternary description
+                (stripTrailing description ".")
+                ""
+              )
+              (ternary (and deprecationReason (not @root.info.x-hideDeprecationReason))
+                (concat "Deprecation Reason: " deprecationReason)
```

Comment:
Here too, can you wrap the reason in a span with a class so styles can be overridden?